spec/spec_helper.rb

```diff
@@ -19,6 +19,7 @@
   end
 
   config.after(:suite) do
+    VIM.command("silent maca hide:")
```

Comment:
None - without the `silent` bit, on Travis, for e.g., it'll complain there's no such command.